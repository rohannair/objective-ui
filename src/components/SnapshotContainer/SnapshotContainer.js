import React, { PropTypes } from 'react'
import styled from 'styled-components'

const SnapshotContainer = styled.div`
  display: flex;
  flex-direction: column;
  background-color: #fff;
  width: 100%;
  padding: 20px 0;
  border: 1px solid #ddd;
  border-top: 0;
  margin-top: 20px;
  box-shadow: #ccc 0 0 5px 1px;
`

export default SnapshotContainer
