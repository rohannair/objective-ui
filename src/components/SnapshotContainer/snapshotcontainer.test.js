import React from 'react'

import SnapshotContainer from './index.js'

describe('Test suite for SnapshotContainer component', () => {
  it('SnapshotContainer should exist', () => {
    let wrapper = shallow(<SnapshotContainer/>)
    expect(wrapper).toMatchSnapshot()
  })
})

