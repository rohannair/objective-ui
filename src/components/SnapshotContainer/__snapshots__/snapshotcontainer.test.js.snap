exports[`Test suite for SnapshotContainer component SnapshotContainer should exist 1`] = `
<div
  className="hJuGYc" />
`;
